containers/elixir-phoenix-postgres/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3"
+
+services:
+  elixir:
+    build: .
```

Comment:
If you use the "VARIANT" idea in your Dockerfile, you could add an `arg:` section that references the VARIANT.